### ISSUE TITLE
Request context

### ISSUE THREAD

**Issue by olegrok** [tarantool/http#111]
As each request is handled in separate fiber it could be useful to store some information in `fiber.storage` (https://github.com/tarantool/tarantool/issues/3462)

**Comment by Totktonada** [comment]
1. There is the [stash](https://github.com/tarantool/http#working-with-stashes). The README shows how to get a captured part of the URL and give a brief API [description](https://github.com/tarantool/http#fields-and-methods-of-the-request-object) without explanatons. In fact, it is inspired by Mojo and works in the similar [way](https://docs.mojolicious.org/Mojolicious/Guides/Routing#Stash): you can store a value here and load it back later during the request processing (from a template as well).
2. You can just use `fiber.storage`.

I'll consider this as the request to improve the documentation.

**Comment by olegrok** [comment]
Yes, my initial thesis is not correct. All requests in the connection is handled by single fiber - it's an implementation detail of HTTP server. It's possible to use just fiber.storage.
